### DISCUSSION TITLE
Error in clickhouse-copier

### DISCUSSION THREAD

**Discussion by wbxsGithub** [ClickHouse/ClickHouse#28061] [Show and tell]
The error occurred in LOG_DEBUG.
There is the log data.
<Debug> ClusterCopier: Executing OPTIMIZE DEDUPLICATE query:  ALTER TABLE xxx.test_for_copier_local ATTACH PARTITION xxx    0 FROM xxx.xxx.

The key is that This statement should execute "OPTIMIZE DEDUPLICATE .......".

**Comment by tavplubix** [comment]
Fixed by https://github.com/ClickHouse/ClickHouse/pull/23518
